torchacc/utils/optim_utils.py

```diff
@@ -0,0 +1,137 @@
+import torch
+import torch.distributed as dist
+from typing import NamedTuple, Optional
+from torch.utils._pytree import tree_map_only
+
+import torch_xla.core.xla_model as xm
+
+
```

Comment:
The functions used in the fsdp.py file do not need to be prefixed with '_'.

Reply:
done